### ISSUE TITLE
Windows: Library build fails

### ISSUE THREAD

**Issue by SwapnilPande** [Roam-Studios/motorgo-mini-driver#107]
```
In file included from .pio/libdeps/spin_two_motors/MotorGo Mini Driver/src/pid_manager.cpp:1:
.pio/libdeps/spin_two_motors/MotorGo Mini Driver/include/pid_manager.h:7:10: fatal error: configurable.h: No such file or directory
```

configurable.h can't be found, despite being present in `.pio/lib_deps`. Including it in the main source file temporarily fixes this issue. This issue does not seem to show up on Linux builds

**Comment by SwapnilPande** [comment]
Found another failing example, while running tests (on Ubuntu 22.04). This appears to be a problem not just OS related.
```
#include <Arduino.h>
#include <unity.h>

#include <vector>

#include "configurable.h"
#include "gcode.h"
#include "motorgo_mini.h"

void gcode_parser_test()
{
  std::vector<String> lines = {
      "G90",
      "G0 X10 Y20 Z30 F100",
      "G1 X20 Y30 Z40 F200",
      "G2 X30 Y40 Z50 F300",
      "G3 X40 Y50 Z60 F400",
      "G28",
      "G90",
  };

  GCode::VectorGCodeStream* stream = new GCode::VectorGCodeStream(lines);
  GCode::GCodeParser* parser = new GCode::GCodeParser(stream);

  GCode::start_parser(*parser);

  if (parser->motion_command_buffer.size() > 0)
  {
    GCode::MotionCommand command = parser->motion_command_buffer.front();
    parser->motion_command_buffer.pop();

    // Print all the motion commands
    Serial.println("X: " + String(command.x));
    Serial.println("Y: " + String(command.y));
    Serial.println("Z: " + String(command.z));
    Serial.println("Feedrate: " + String(command.feedrate));
    Serial.println("Home: " + String(command.home));
    delay(100);
  }

  delete stream;
}

void setup()
{
  Serial.begin(115200);
  gcode_parser_test();
}

void loop()
{
  // Nothing to do here
}
```

Code to reproduce this. Removing `#include "configurable.h"` causes build to fail.

**Comment by SwapnilPande** [comment]
PIO Version 6.1.15 seems to have fixed the bug that causes this issue. We should verify on all platforms and make sure the library builds correctly now.

**Comment by SwapnilPande** [comment]
Changes included in V1.0.2 release
